_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,465 @@
+            <label class="field-label label-inline" for="updates_email_04">
+                <span class="field-input-label">Please send me a handwritten letter (unnested label and input)</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="updates_email_04">
+                <input type="radio" class="field-input input-radio" name="updates_notifications_01" id="updates_email_03" value="Email messages" disabled>
+                <span class="field-input-label">Never contact me (disabled radio button)</span>
+            </label>
+        </div>
+    </fieldset>
+    <fieldset class="field-group">
+        <legend class="field-group-hd field-label">How would you prefer to be contacted? (enhanced radio buttons)</legend>
+        <div class="field">
+            <label class="field-label label-inline" for="updates_mobile_text_02">
+                <input type="radio" class="field-input input-radio replace-radio" name="updates_notifications_02" id="updates_mobile_text_02" value="Mobile text messages" aria-describedby="updates_mobile_text_02_helper" checked="checked">
+                <span class="field-input-label">Text messages on my mobile phone</span>
+            </label>
+            <div class="field-hint" id="updates_mobile_text_02_helper">Rates apply. Pelase consult your plan.</div>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="updates_email_02">
+                <input type="radio" class="field-input input-radio replace-radio" name="updates_notifications_02" id="updates_email_02" value="Email messages" aria-describedby="updates_email_02_helper">
+                <span class="field-input-label">Email at matttracker@gmail.com</span>
+            </label>
+            <div class="field-hint" id="updates_email_02_helper">We will never spam you. Pinky swear.</div>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="updates_email_04">
+                <input type="radio" class="field-input input-radio replace-radio" name="updates_notifications_02" id="updates_email_04" value="Email messages" disabled>
+                <span class="field-input-label">Never contact me (disabled radio button)</span>
+            </label>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Checkboxes</h4>
+<div class="example-set">
+    <fieldset class="field-group">
+        <legend class="field-group-hd field-label">Select your favorite pizza toppings (default checkboxes)</legend>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_01">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_01" value="Pepperoni" checked="checked">
```

Comment:
NIT: Let's keep this consistent with our attribute standards (and what you're doing with required fields/attrs) by just using the ``checked`` notation here. Cool?